tests/conftest.py

```diff
@@ -103,106 +94,89 @@ def gov(accounts):
-# @pytest.fixture(scope="module")
-# def LiveStrategy_1():
-#     yield Contract("0xC1810aa7F733269C39D640f240555d0A4ebF4264")
+# use this if your vault is already deployed
+# @pytest.fixture(scope="function")
+# def vault(pm, gov, management):
+#     vault = pm(config["dependencies"][0]).Vault
+#     vault = vault.at("0xFfe9fa48A805AC26eEF9DC750765C4dFB530f70b")
+#     vault.setGovernance(gov, {"from": management})
+#     vault.acceptGovernance({"from": gov})
+#     vault.setManagementFee(200, {"from": gov})
+#     vault.setPerformanceFee(2000, {"from": gov})
+#     yield vault
 
 
-# use this if you need to deploy the vault
+# use a fresh vault
 @pytest.fixture(scope="function")
-def vault(pm, gov, rewards, guardian, management, token, chain, xdai_whale):
+def vault(pm, gov, rewards, strategist, management, token, chain):
     Vault = pm(config["dependencies"][0]).Vault
-    xdai_whale.transfer(gov, "1 ether")
-    vault = guardian.deploy(Vault)
-    vault.initialize(token, gov, rewards, "", "", guardian, {'from':gov})
-    vault.setDepositLimit(2 ** 256 - 1, {"from": gov})
+    vault = strategist.deploy(Vault)
+    vault.initialize(token, gov, rewards, "", "", strategist)
+    vault.setDepositLimit(Wei("50_000 ether"), {"from": gov})
     vault.setManagement(management, {"from": gov})
-    chain.sleep(1)
+    vault.setManagementFee(200, {"from": gov})
+    vault.setPerformanceFee(2000, {"from": gov})
     yield vault
 
 
-# use this if your vault is already deployed
-# @pytest.fixture(scope="function")
-# def vault(pm, gov, rewards, guardian, management, token, chain):
-#     vault = Contract("0x497590d2d57f05cf8B42A36062fA53eBAe283498")
-#     yield vault
-
-
 # replace the first value with the name of your strategy
 @pytest.fixture(scope="function")
 def strategy(
     StrategyCurve3crv,
     strategist,
-    keeper,
     vault,
     gov,
-    guardian,
-    token,
-    healthCheck,
-    chain,
-    pool,
     strategy_name,
-    gauge,
-    strategist_ms,
-    xdai_whale
+    other_vault_strategy,
+    xdai_whale,
+    tradeFactory,
 ):
     # make sure to include all constructor parameters needed here
     xdai_whale.transfer(strategist, "1 ether")
-    xdai_whale.transfer(strategist_ms, "1 ether")
-    strategy = strategist.deploy(
-        StrategyCurve3crv,
+
+    strategy = StrategyCurve3crv.deploy(
         vault,
         strategy_name,
+        {"from": strategist},
     )
 
-    strategy.setKeeper(keeper, {"from": gov})
-    # set our management fee to zero so it doesn't mess with our profit checking
-    vault.setManagementFee(0, {"from": gov})
     # add our new strategy
-    vault.addStrategy(strategy, 10_000, 0, 2 ** 256 - 1, 1_000, {"from": gov})
-    strategy.setHealthCheck(healthCheck, {"from": gov})
+    vault.addStrategy(strategy, 10_000, 0, 2**256 - 1, 0, {"from": gov})
+    # other_vault_strategy.harvest({"from": gov})
+    # vault.migrateStrategy(other_vault_strategy, strategy, {"from": gov})
```

Comment:
in line with the comment above, note that we add a new blank strategy as opposed to migrating the old one. The assets and accounting of the old strategy messed up a few tests.